### ISSUE TITLE
2.016 broken, and 2.016 beta doesn't even work

### ISSUE THREAD

**Issue by fearthedice** [Aktanusa/CookieMonster#225]
error happens both on the beta and the non- beta(CookieMonster/CookieMonster.js and CookieMonster/CookieMonsterBeta.js)
![capture](https://user-images.githubusercontent.com/44506942/47582987-ff977980-d923-11e8-8ac4-83abdbbf1f0c.PNG)
PS: i am using book mark but have tried tampermonkey and console, neither work, although, if i comment out the part that gives an error, it actually loads, but payback time is broken and only working thing is golden cookies. that works.

**Comment by Aktanusa** [comment]
It be nice if you took at look at Issue #223 and Issue #224 first...

**Comment by fearthedice** [comment]
i checked them, but it looked like that was before the 2.016 release. sorry if i messed up though.

**Comment by Aktanusa** [comment]
Issue #224 says 2.016 in the title.  Oh well, it's fine.  I know everyone really wants this to be fixed, but please be patient.

**Comment by Stanzilla** [comment]
Tried your latest commit, getting 
``` 
Uncaught TypeError: Cannot read property 'toString' of undefined at CookieMonster.js:2911
```

on 
`eval('CM.Sim.getGrandmaSynergyUpgradeMultiplier = ' + Game.getGrandmaSynergyUpgradeMultiplier.toString().split('Game.Objects[\'Grandma\']').join('CM.Sim.Objects[\'Grandma\']'));`

**Comment by Aktanusa** [comment]
Thanks!  I should note the latest commit only says I did a version change.  I've not finished fixing yet.

**Comment by Stanzilla** [comment]
Yeah no worries, just wanted to post about it :)

**Comment by Aktanusa** [comment]
Partially fixed in Version 2.016.1, now to do the beta.

**Comment by Aktanusa** [comment]
Beta is also fixed to Version 2.016.1
